### PR TITLE
test: resolve integration test failures due to insufficient test tokens

### PR DESCRIPTION
**Pull request by Dawn-githup**
### Main Changes:

- Switched from Signet to Testnet3 for testing due to insufficient Signet test resources
- Added pull_request trigger for tests
- Updated btcConfirmationBlocks in btc-leap-ckb to 20 instead of using the default value of 6"

### CODE REVIEW

File: .github/workflows/integration-test.yaml [hunk 1]
```diff
@@ -4,6 +4,7 @@ name: Integration Tests
 
 on:
   workflow_dispatch:
+  pull_request:
```
Comment (Flouse) [inline .github/workflows/integration-test.yaml:7]: LGTM.

If you want save some tBTC, you might consider [running a pull_request_target workflow when a pull request merges](https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#running-your-pull_request_target-workflow-when-a-pull-request-merges).

**Review by Flouse — DISMISSED** [review]

File: .github/workflows/integration-test.yaml (hunk 1 above)
Comment (Dawn-githup) [reply .github/workflows/integration-test.yaml:7]: > LGTM.
> 
> If you want save some tBTC, you might consider [running a pull_request_target workflow when a pull request merges](https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#running-your-pull_request_target-workflow-when-a-pull-request-merges).

We have enough tBTC available, so we can conduct tests early.

File: tests/rgbpp/xudt/3-btc-leap-ckb.ts [hunk 2]
```diff
@@ -29,6 +33,7 @@ const leapFromBtcToCKB = async ({ rgbppLockArgsList, toCkbAddress, xudtTypeArgs,
       toCkbAddress,
       isMainnet,
       btcTestnetType: BTC_TESTNET_TYPE,
+      btcConfirmationBlocks: 20, // default value is 6
```
Comment (duanyytop) [inline tests/rgbpp/xudt/3-btc-leap-ckb.ts:36]: The normal leaping from BTC to CKB is 6 confirmations, not 20. The test case is for normal leaping not for Seal2earn.

Comment (Dawn-githup) [reply tests/rgbpp/xudt/3-btc-leap-ckb.ts]: > The normal leaping from BTC to CKB is 6 confirmations, not 20. The test case is for normal leaping not for Seal2earn.

https://github.com/ckb-cell/rgbpp-sdk/pull/262/commits/5599bedd9feb23ab8932b3507eec8a405deee357

**Review by Flouse — APPROVED** [review]

**Review by duanyytop — APPROVED** [review]

**Pull request merged**